sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.c

```diff
@@ -439,6 +437,15 @@ void _set_reachable(gnrc_netif_t *netif, _nib_onl_entry_t *nce)
                  netif->ipv6.reach_time);
 }
 
+void _set_unreachable(gnrc_netif_t *netif, _nib_onl_entry_t *nce)
+{
+    DEBUG("nib: set %s to UNREACHABLE\n",
+          ipv6_addr_to_str(addr_str, &nce->ipv6, sizeof(addr_str)));
+
+    _nbr_flush_pktqueue(nce);
```

Comment:
Forgot to guard this with `#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_QUEUE_PKT)`.

`USE_ZEP=1 make -C examples/gnrc_networking` fails